src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,29 @@
-@mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
+@mixin on744px {
+  @media (max-width: 1020px) {
+    @content;
   }
 }
+
+@mixin on320px {
+  @media (max-width: 744px) {
+    @content;
+  }
+}
+
+@mixin gridOn1024px {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 24px;
+}
+
+@mixin gridOn744px {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  gap: 24px;
+}
+
+@mixin gridOn320px {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 24px;
+}
```

Comment:
Better move all your 'grid' code to `grid.scss` file.

Detailed video  [heve](https://mate.academy/learn/css-advanced/css-advanced-landing#/video/1310)